NitroxClient/GameLogic/EquipmentSlots.cs

```diff
@@ -58,15 +70,8 @@ public void BroadcastUnequip(Pickupable pickupable, GameObject owner, string slo
         }
         else
         {
-            // Reactor rod can't be unequipped so this will only happen when a Nuclear Reactor is destroyed (in which case we don't need this code)
-            if (pickupable.GetTechType() == TechType.ReactorRod)
-            {
-                return;
-            }
-
             // UWE also sends module events here as they are technically equipment of the vehicles.
```

Comment:
As we don't discriminate that hard anymore between modules and items I would remove this comment (same for `BroadcastEquip()`). Or does "module events" mean something special?